src/scss/components/eyebrow/index.scss

```diff
@@ -0,0 +1,126 @@
+// ----- BASE ELEMENTS ---------------
+
+.eyebrow {
+  overflow: hidden;
+  width: 100%;
+  height: 56px;
+  background-color: #000000;
+}
+
+.img-left {
+  float: left;
+  width: 159px;
+  height: 56px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.right-menu {
+  float: right;
+}
+
+.eyebrow a img {
+  width: 119px;
+  height: 24px;
+}
+
+.eyebrow a {
+  float: left;
+  font-size: 13px;
+  color: #ffffff;
+  text-align: center;
+  padding: 1rem 9px;
+  text-decoration: none;
+  line-height: 1;
+  height: 56px;
+}
+
+.eyebrow p {
+  color: #ffffff;
+  line-height: 1.5;
+}
+
+.dropdown {
+  float: right;
+  overflow: hidden;
+}
+
+.dropdown .dropbtn {
+  font-size: 12px;
+  border: none;
+  outline: none;
+  color: #ffffff;
+  padding: 14px 16px;
+  background-color: inherit;
+  font-family: inherit;
+  margin: 0;
+  height: 56px;
+}
+
+.eyebrow a:hover, .dropdown:hover .dropbtn {
+  background-color: #242424;
+}
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #242424;
```

Comment:
Please update to use HelixUI color vars:

https://github.com/HelixDesignSystem/helix-ui/blob/master/src/scss/_vars.scss